### ISSUE TITLE
App not accessible for the screen reader users on Mac

### ISSUE THREAD

**Issue by aureliomuth** [langerhans/multidoge#109]
Hi,
I wish to use the Dogecoin core On my Mac but it is unaccessible for the Voiceover screen reader. Can you do something? Thank you very much!
